rust_tonic_mt_bench/Cargo.toml

```diff
@@ -8,6 +8,7 @@ edition = "2018"
 opt-level = 3
 codegen-units = 1
 lto = true
+debug = true
```

Comment:
Why do we need this change?

Reply:
We don't, not sure how it snuck in there! Good catch!